posts/2021-08-28-AndyMcCright_MeetTheTeam.adoc

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: "Meet our open source champions: Andy McCright, Web Services Architect"
+title: "Meet our open source champions- Andy McCright, Web Services Architect"
 # Do NOT change the categories section
 categories: blog
-author_picture: https://avatars3.githubusercontent.com/andymc12 
-author_github: https://github.com/andymc12 
+author_picture: https://avatars3.githubusercontent.com/andymc12
+author_github: https://github.com/andymc12
 seo-title: Meet our open source champion, Andy McCright, Web Services Architect
-seo-description: 
-blog_description: 
-open-graph-image: https://openliberty.io/img/twitter_card.jpg
+seo-description: Andy is a Web Services Architect who enjoys open source software and contributes to many projects, including Open Liberty, Apache CXF, RESTEasy, Jakarta RESTful Web Services, and MicroProfile.
+blog_description: Andy is a Web Services Architect who enjoys open source software and contributes to many projects, including Open Liberty, Apache CXF, RESTEasy, JAX-RS, and MicroProfile.
+open-graph-image: https://openliberty.io/img/AndyMcCright-1.png
```

Comment:
This link doesn't work - I get a 404.  Should the image be uploaded with this PR?

Reply:
the open graph image wont work until the post goes to prod. The image is the same as the first one in the post, which looks good.